### ISSUE TITLE
YouTube video not playing in Reveal JS

### ISSUE THREAD

**Issue by rressler** [quarto-dev/quarto-cli#4089]
Trying to embed a YouTube video in a reveal JS presentation.

Have tried using the shortcode for video with several variants of the URL for the video, and the embed code such as the following:

`{{< video https://www.youtube.com/watch?v=Ye8mB6VsUHw&t=16s width="850" height="475">}}` produces nothing.

`{{< video https://www.youtube.com/embed/Ye8mB6VsUHw width="850" height="475">}}` and `{{< video https://youtu.be/Ye8mB6VsUHw width="850" height="475">}}`

produce a black frame. When I inspect the frame in the browser I see the following with a long set of characters data where the ... are in the following for the src

```
<iframe src="data:text/html; ......................... .charset=utf-8;charset=utf-8, width="850" height="475" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```

If I edit the element in the browser to replace the src as follows,  

```
<iframe src="https://www.youtube.com/embed/Ye8mB6VsUHw" width="250" height="175" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```
the video runs fine.

What am I missing to get it to render so it works?

Appreciate any suggestions.

**Comment by cscheid** [comment]
@rressler What version of quarto are you using? I can't reproduce what you're seeing, and I made a fix about this on 1.3.124: https://github.com/quarto-dev/quarto-cli/commit/339543137c6d19257963a51908ed1ebbca839b6d

**Comment by rressler** [comment]
Carlos,
I was using the deployed version so I updated to the development version and restarted RStudio .
Now it works on all the versions below. Nicely done.

Thanks so much!
Richard

{{< video https://www.youtube.com/watch?v=Ye8mB6VsUHw&t=2s width="850" height="475">}}

{{< video https://www.youtube.com/embed/Ye8mB6VsUHw width="850" height="475">}}

{{< video https://youtu.be/Ye8mB6VsUHw width="850" height="475">}}
From: Carlos Scheidegger ***@***.***>
Date: Tuesday, January 24, 2023 at 7:49 PM
To: quarto-dev/quarto-cli ***@***.***>
Cc: Richard Ressler ***@***.***>, Mention ***@***.***>
Subject: Re: [quarto-dev/quarto-cli] YouTube video not playing in Reveal JS (Issue #4089)

External Email: Use caution with links and attachments.

@rressler<https://urldefense.com/v3/__https:/github.com/rressler__;!!IaT_gp1N!2EYa7XIulzVkd_nMH_FUhZtSqndSxjKK0JRlAA32yj1t8H2jb5fLbeolq7GI8_dcDfpeIc5kSbac7sUBUzqn61R3kS4$> What version of quarto are you using? I can't reproduce what you're seeing, and I made a fix about this on 1.3.124: 3395431<https://urldefense.com/v3/__https:/github.com/quarto-dev/quarto-cli/commit/339543137c6d19257963a51908ed1ebbca839b6d__;!!IaT_gp1N!2EYa7XIulzVkd_nMH_FUhZtSqndSxjKK0JRlAA32yj1t8H2jb5fLbeolq7GI8_dcDfpeIc5kSbac7sUBUzqngpNDj1Q$>

**Comment by allenmanning** [comment]
Thank you Carlos!
